packages/dita-example-sx-modules-xsd-common-element-mod/src/install.js

```diff
@@ -0,0 +1,28 @@
+import documentsManager from 'fontoxml-documents/src/documentsManager.js';
+import addAction from 'fontoxml-operations/src/addAction.js';
+import remoteDocumentStateManager from 'fontoxml-remote-documents/src/remoteDocumentStateManager.js';
+
+export default function install() {
+	addAction(
+		'isOutOfSyncDocument',
+		function isOutOfSyncDocument(stepData) {},
```

Comment:
The execute callback for this action should also be able to cancel the operation in cases where the operation is executed without first checking its state.